src/Compilers/CSharp/Test/Symbol/Symbols/MissingSpecialMember.cs

```diff
@@ -623,6 +623,7 @@ public void AllWellKnownTypes()
                     case WellKnownType.System_Runtime_CompilerServices_ITuple:
                     case WellKnownType.System_Runtime_CompilerServices_NonNullTypesAttribute:
                     case WellKnownType.Microsoft_CodeAnalysis_EmbeddedAttribute:
+                    case WellKnownType.System_GC:
```

Comment:
Was that necessary? The `System.GC` type existed pretty far back (desktop 4.6, netstandard 2.0, and probably further back)

Reply:
It actually didn't already exist as a well-known type. I was surprised about that too

Reply:
> It actually didn't already exist as a well-known type. I was surprised about that too

Julien specifically meant did this test need updating? `System.GC` likely exists on all the platforms this test runs against.

Reply:
Ah I see my point of confusion now. The `AllWellKnownTypeMembers` tests *did* need updating, because `System.GC.AllocateUninitializedArray` does not exist as a member in all of the assemblies that are part of this test. But Julien is absolutely right that `AllWellKnownTypes` does not need updating, because `System.GC` does exist in all of the assemblies.